src/megaapi_impl.cpp

```diff
@@ -8726,20 +8721,7 @@ void MegaApiImpl::sendPendingRequests()
                     attrvalue = value;
                     break;
                 }
-
-                case MegaApi::USER_ATTR_AUTHRING:
-                {
-                    attrname = "*!authring";
-                    attrvalue = value;
-                    break;
-                }
-
-                case MegaApi::USER_ATTR_LAST_INTERACTION:
-                {
-                    attrname = "*!lstint";
-                    attrvalue = value;
-                    break;
-                }
+                // authentication ring and last interaction are ro attributes
```

Comment:
Why is that? They can be (and are) modified frequently. At least on the webclient.